src/extensions/extension_manager/inc/aduc/extension_manager_download_options.h

```diff
@@ -3,7 +3,8 @@
 
 typedef struct tagADUC_ExtensionManager_Download_Options
 {
-    unsigned int retryTimeout;
+    unsigned int
+        maxTimeoutInSeconds; /**< The maximum number of seconds the content downloader should try to keep downloading. */
```

Comment:
is seconds the best unit to use here?  do we feel a customer would decide to use 63 seconds vs 127 seconds?

Seems like minutes is more logical.